sdk/cosmos/src/clients/collection_client.rs

```diff
@@ -137,4 +157,18 @@ impl CollectionClient {
     pub(crate) fn http_client(&self) -> &dyn HttpClient {
         self.cosmos_client().http_client()
     }
+
+    pub(crate) fn pipeline(&self) -> &Pipeline<CosmosContext> {
+        self.cosmos_client().pipeline()
+    }
+
+    fn prepare_request_pipeline(&self, http_method: http::Method) -> Request {
+        let path = &format!(
+            "dbs/{}/colls/{}",
```

Comment:
This should be `"dbs/{}/colls/{}/docs"` as per [https://docs.microsoft.com/en-us/rest/api/cosmos-db/create-a-document](https://docs.microsoft.com/en-us/rest/api/cosmos-db/create-a-document)